app/src/features/projects/create/CreateProjectPage.tsx

```diff
@@ -66,7 +66,7 @@ const CreateProjectPage: React.FC = () => {
 
   // Reference to pass to the formik component in order to access its state at any time
   // Used by the draft logic to fetch the values of a step form that has not been validated/completed
-  const formikRef = useRef<FormikProps<ICreateProjectRequest>>(null);
+  const [formikRef] = useState(useRef<FormikProps<ICreateProjectRequest>>(null));
```

Comment:
Is there a way around creating a state variable that never gets updated? To me it seems like a bit of an anti-pattern

Reply:
Good question.  I agree that it is odd, however I suspect that the reason we have this pattern established throughout the code isthat formik.current.values is constantly updated as users type in the form.  Do you have suggestions for a different setup?

Reply:
If we're hooking into useState because we want to preserve the value between renders, we can use the useMemo hook instead 🤔 I'll try pulling the branch and see what I can come up with

Reply:
hmm interesting, I'll check out useMemo too

Reply:
@curtisupshall ... turns out the Usestate was unneccessary, as the useRef itself is sufficient.  What we needed to do was check that formik.current is not null before setting the values .. and this solution doesn't have the added complexity of a useState or Use Memo.  

I also looked into useMemo, it doesn't apply to this case, because the intention of useMemo to memoize computational heavy components ... and I don't think the draft functionality is heavy ... but maybe loading the whole project is.  

Either way, seems to work fine now.  Would you please take a look again when you have a chance?